E.Deezer/E.Deezer.csproj

```diff
@@ -85,11 +85,9 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="RequestParameter.cs" />
   </ItemGroup>
-  <ItemGroup />
   <ItemGroup>
-    <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\packages\Newtonsoft.Json.10.0.3\lib\portable-net45+win8+wp8+wpa81\Newtonsoft.Json.dll</HintPath>
-      <Private>True</Private>
+    <Reference Include="Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
I'm not sure about this. I'd rather keep Newtonsoft at the 10.0.0 range. Iirc Newtonsoft stopped supporting PCL and moved .NET Standard in 11.0.0+. .NET Standard support is on my list of things to do if I ever get around to doing it. Thoughts?

Reply:
I use the lib for my ASP.NET Core project, that's why I have updated the Newtonsoft.Json.
I know really nothing about PCL ... :-)